reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -79,18 +92,35 @@ export default function WorkflowList({ workflows, refreshedAt }) {
             </div>
             <div className={styles["status-box"]}>
               <span
-                className={`${styles["status"]} sui-${statusMapping[status].color}`}
+                className={`${styles.status} sui-${statusMapping[status].color}`}
               >
                 {status}
               </span>{" "}
-              {statusMapping[status].preposition} {duration}
+              {statusMapping[status].preposition}{" "}
+              {status !== "deleted" && duration}
               <div>
                 step {completed}/{total}
               </div>
             </div>
+            <div className={styles.actions}>
+              {(status !== "deleted" ||
+                (status === "deleted" && size !== "0K")) && (
+                <WorkflowActionsPopup
```

Comment:
I noticed that it's very easy to miss the button when you are trying to press it. Then and you are being redirected to `workflow details` page. So I wonder if we could increase the margin of the button, so it could have a bit bigger area

Reply:
Addressed.